lib/3scale/api/client.rb

```diff
@@ -103,6 +103,15 @@ def create_service(attributes)
 
       # @api public
       # @return [Hash]
+      # @param [Hash] attributes Service Attributes
+      # @option attributes [String] :name Service Name
+      def update_service(attributes, service_id)
+        response = http_client.put("/admin/api/services/#{service_id}", body: { service: attributes })
```

Comment:
The documented endpoint for Service Update is using `PUT`:
![screen shot 2017-07-10 at 18 20 02](https://user-images.githubusercontent.com/1270328/28028322-7125bd70-659c-11e7-8e39-3ffcf35b1fed.png)